### PR TITLE
Force slack posts locale

### LINKED ISSUE #361: Language in slack interaction
A new post will show up in slack with the language of the submitter because the url generated by `post_url` will include the locale of the submitter.

### PR DESCRIPTION
**Pull request by malmers**
Force slack news locale to Swedish

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -144,6 +144,6 @@ def send_irkk
 
     def send_slack
       notifier = Slack::Notifier.new Rails.application.secrets.slack_url
-      notifier.post unfurl_links: true, unfurl_media: true, text: "New post published: *[#{@post.title}](#{post_url(@post)})* by #{@post.user.display_name}"
+      notifier.post unfurl_links: true, unfurl_media: true, text: "New post published: *[#{@post.title}](#{post_url(@post, locale: 'sv')})* by #{@post.user.display_name}"
```
Comment (lindskogen) [inline app/controllers/posts_controller.rb]: `@post.title` needs to be fetched in `sv` as well right? `@post.title_sv` should be enough :)

**Comment by lindskogen** [comment]
@malmers are you planning to finish this? :)

**Comment by malmers** [comment]
Yes, any day now

**Comment by Hoidi** [comment]
@Portals do you still not want to merge this 😁

**Comment by lindskogen** [comment]
@Portals why no merge?

**Comment by Portals** [comment]
I think it was because it wasn't finished yet. But if it's now then I will remove it

**Pull request merged**